lib/spandex_datadog/api_server.ex

```diff
@@ -30,6 +30,9 @@ defmodule SpandexDatadog.ApiServer do
     ]
   end
 
+  # Same as HTTPoison.headers
+  @type headers :: [{atom, binary}] | [{binary, binary}] | %{binary => binary} | any
```

Comment:
I copied this type definition from HTTPoison instead of referencing, since it's an optional / test-only dependency.

Reply:
That was a good decision :)